hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/SubProcessHapiTestNode.java

```diff
@@ -284,15 +285,16 @@ public void clearState() {
         }
 
         final var saved = workingDir.resolve("data/saved").toAbsolutePath().normalize();
+        Stream<Path> paths = null;
         try {
             if (Files.exists(saved)) {
-                Files.walk(saved)
-                        .sorted(Comparator.reverseOrder())
-                        .map(Path::toFile)
-                        .forEach(File::delete);
+                paths = Files.walk(saved);
+                paths.sorted(Comparator.reverseOrder()).map(Path::toFile).forEach(File::delete);
             }
         } catch (IOException e) {
             throw new RuntimeException("Could not delete saved state " + saved, e);
+        } finally {
+            if (paths != null) paths.close();
         }
```

Comment:
Try this (github won't let me do a suggestion here):
```java
        if (Files.exists(saved)) {
            try(paths = Files.walk(saved)) {
                paths.sorted(Comparator.reverseOrder()).map(Path::toFile).forEach(File::delete);
            } catch (IOException e) {
                throw new RuntimeException("Could not delete saved state " + saved, e);
            }
        }
```

Reply:
done thanks